### DISCUSSION TITLE
New Author URL

### DISCUSSION THREAD

**Discussion by enikad13** [Videogamers0/StoryManager#18] [Ideas]
Wasn't really sure where to put this, but I noticed Literotica has changed the URL for authors.  It's now in the format of https://www.literotica.com/authors/banditray5/works/stories and StoryManager doesn't recognize it.

**Comment by Videogamers0** [comment]
33b941011e00565a769223ee77eb777c6892f1bd

Fixed in [version 1.0.7](https://github.com/Videogamers0/StoryManager/releases/latest)

**Reply by enikad13** [reply]
awesome. thanks for doing this.  kinda funny because i had just updated to v1.0.6 shortly before posting (was told it was the latest version).
